src/main/scala/de/upb/cs/swt/delphi/instanceregistry/RequestHandler.scala

```diff
@@ -804,6 +804,56 @@ class RequestHandler(configuration: Configuration, connection: DockerConnection)
     }
   }
 
+  /**
+    * Handles a call to /command. container id and command must be present,
+    * Will run the command into the container with provide parameters
+    *
+    * @param id container id the command will run on
+    * @param command the command to run
+    * @param attachStdin attaches to stdin of the command
+    * @param attachStdout attaches to stdout of the command
+    * @param attachStderr attaches to stderr of the command
+    * @param detachKeys  Override the key sequence for detaching a container.
+    *                    Format is a single character [a-Z] or ctrl-<@value> where <v@alue> is one of: a-z, @, [, , or _
+    * @param privileged  runs the process with extended privileges
+    * @param tty  allocate a pseudo-TTY
+    * @param user A string value specifying the user, and optionally, group to run the process inside the container,
+    *             Format is one of: "user", "user:group", "uid", or "uid:gid".
+    * @return
+    */
+  def handleCommand(id: Long, command: String, attachStdin: Option[Boolean],
+                    attachStdout: Option[Boolean],
+                    attachStderr: Option[Boolean],
+                    detachKeys: Option[String],
+                    privileged: Option[Boolean],
+                    tty: Option[Boolean],
+                    user: Option[String]): OperationResult.Value = {
+    if (!instanceDao.hasInstance(id)) {
+      OperationResult.IdUnknown
+    } else if (!isInstanceDockerContainer(id)) {
+      OperationResult.NoDockerContainer
+    } else {
+      val instance = instanceDao.getInstance(id).get
+      log.info(s"Handling /command for instance with id $id...")
+      implicit val timeout : Timeout = Timeout(10 seconds)
+
+      val future: Future[Any] = dockerActor ? runCommand(instance.dockerId.get, command, attachStdin, attachStdout, attachStderr, detachKeys, privileged, tty, user)
+      val commandRunResult = Await.result(future, timeout.duration).asInstanceOf[Try[(String, String, Int)]]
```

Comment:
Executing a command is a potentially long-runnning operation. ```Await.result``` will block the execution until it is finished, which is not good style for HttpRequests. Please make the operation is run in the background, like it is done for most of the other operations. Here is an example for start:
```
        log.info("Starting container...")
        implicit val timeout: Timeout = configuration.dockerOperationTimeout

        (dockerActor ? start(instance.dockerId.get)).map{
          _ => log.info(s"Instance $id started.")
        }.recover {
          case ex: Exception =>
            log.warning(s"Failed to start container with id $id. Message is: ${ex.getMessage}")
            fireDockerOperationErrorEvent(Some(instance), s"Start failed with message: ${ex.getMessage}")
        }

        OperationResult.Ok
```